lxd/tokens.go

```diff
@@ -55,7 +62,7 @@ func autoRemoveExpiredTokens(ctx context.Context, s *state.State) {
 		for _, op := range expiredTokenOps {
 			_, err := op.Cancel()
 			if err != nil {
-				logger.Warn("Failed removing expired token", logger.Ctx{"err": err, "operation": op.ID()})
+				logger.Debug("Failed removing expired token", logger.Ctx{"err": err, "operation": op.ID()})
```

Comment:
why not Warn?

Reply:
Hmm this is just wrong, it shouldn't have been changed in this commit. Will fix.

Reply:
Fixed.